content/articles/create-screen-recorder-in-javascript/index.md

```diff
@@ -0,0 +1,219 @@
+title: Section Engineering Education
+
+description: Resources created by engineers for engineers
+
+aliases:
+  - '/create-screen-recoder/'
+  - '/javascript-screen-recorder/'
+type: articles
+
+images:
+  - url: /engineering-education/create-screen-recorder-in-javascript/hero.jpg
+  
+    alt: Screen Recorder in JavaScript
+
+
+### Introduction
+The Screen Capture API is used to capture the screen or a part of the screen as a Media Stream. From the recorded Media Stream we can create a video file.
+
+### Prerequisites
+
+To follow along with this tutorial, you should have:
+
+-   A basic understanding of HTML and JavaScript.
+
+-   A code editor. I’ll be using  [Visual Studio Code](https://code.visualstudio.com/download).
+
+-   A browser to view the webpage, preferably  [Google Chrome](https://www.google.com/intl/en_in/chrome/).
+
+### Project directory
```

Comment:
I love how you have covered the prerequisites. 
Good job!
Next, for a tutorial, we normally work with steps. So for instance I assume the project directory is a step what you will do is write:
### Step one: Creating a new project directory.
In this step...

Do this for all the steps. Remember to also have well-written headings.

Reply:
i have changed this